### ISSUE TITLE
Tables

### ISSUE THREAD

**Issue by teamrehab** [andlabs/ui#290]
Any idea when tables and menubar will be brought to UI?

Thanks!

**Comment by andlabs** [comment]
No timeframe, sorry. Tables themselves are not yet in libui, and I want to redo menus.

Duplicate of #277

**Comment by undeadindustries** [comment]
Sorry, second part of this question. I have bcampell/ui tables branches checked out. When I try to run it, I get: cgo-gcc-prolog:66: undefined reference to `uiNewTable'

Do I have to download the table branch of libgui/

**Comment by andlabs** [comment]
I don't know, but anything on that branch will not apply to this once that branch is merged.

Locking in favor of duplicate.
